easybuild/easyconfigs/g/GCCcore/GCCcore-5.3.0.eb

```diff
@@ -15,7 +15,7 @@ source_urls = [
     'http://ftpmirror.gnu.org/gnu/gcc/gcc-%(version)s',  # GCC auto-resolving HTTP mirror
     'http://ftpmirror.gnu.org/gnu/gmp',  # idem for GMP
     'http://ftpmirror.gnu.org/gnu/mpfr',  # idem for MPFR
-    'http://ftpmirror.gnu.org/gnu/mpc',  # idem for MPC
+    'http://www.multiprecision.org/mpc/download',  # MPC official
```

Comment:
This URL is not correct anymore, we changed this in #5662.

Please sync your branch with current `develop`?

Reply:
Do I need to run full `develop` EasyBuild for this, or does it suffice to manipulate the repo in `git-working-dirs-path`?

Reply:
You don't need to run full `develop` EasyBuild to do that, but you do need to use `git` commands currently (check out the current `develop` branch, `git merge develop` into the branch for your PR, `git push` the updated branch). We should probably look into `eb --sync-pr-develop` or something...

I'll send a PR your way to handle this for now.

Reply:
Great, I'm in the process of addressing the feedback and figuring out where my patches came from.

Reply:
@zao issues with MPC source URLs fixed in https://github.com/zao/easybuild-easyconfigs/pull/1

That PR is going to be impossible to review, but if you want to review it, just look at https://github.com/zao/easybuild-easyconfigs/pull/1/commits/b83c5e849baa000c85832c385a31d05d12a01e75